Tests/CreateAPITests/Snapshots.swift

```diff
@@ -5,26 +5,27 @@
 import Foundation
 import XCTest
 
-// TODO: Find it automatically
-var projectPath = ("~/Developer/CreateAPI/" as NSString).expandingTildeInPath
-
 func compare(expected: String, actual: String, file: StaticString = #file, line: UInt = #line) throws {
     let env = ProcessInfo.processInfo.environment
     let expectedURL = Bundle.module.resourceURL!
         .appendingPathComponent("Expected")
         .appendingPathComponent(expected)
     let actualURL = URL(fileURLWithPath: actual)
+    
     if env["GENERATE_SNAPSHOTS"] == "true" || !FileManager.default.fileExists(atPath: expectedURL.path) {
-        let projectPath = (projectPath as NSString).expandingTildeInPath
-        let destinationURL = URL(fileURLWithPath: projectPath + "/Tests/CreateAPITests/Expected/\(actualURL.lastPathComponent)")
+        let testsFolderPath = #filePath
+            .split(separator: "/")
+            .dropLast()
+            .joined(separator: "/")
+        let destinationURL = URL(fileURLWithPath: "/" + testsFolderPath + "/Expected/\(actualURL.lastPathComponent)")
```

Comment:
I'm just wondering if we can go one step further and create `URL(fileURLWithPath: #filePath)` and then use `appendingPathComponent(_:)`/`deletingLastPathComponent()` to construct the final path instead of relying on string manipulation directly? 

```swift
let destinationURL = URL(fileURLWithPath: #filePath)
    .deletingLastPathComponent()
    .appendingPathComponent("Expected")
    .appendingPathComponent(actualURL.lastPathComponent)
```